.github/workflows/nix.yml

```diff
@@ -7,11 +7,7 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - uses: cachix/install-nix-action@v12
-      with:
-        nix_path: nixpkgs=channel:nixos-unstable
     - uses: cachix/cachix-action@v8
       with:
         name: nix-community
-        extraPullNames: nixpkgs-update
-        skipPush: true
```

Comment:
Is `skipPush` implied by not having the secrets provided?

Reply:
good question. I pushed an update that sets the signingKey instead.

Reply:
This was originally using your numtide branch with the name "only-cache". I think we wanted to skip pushing to avoid pushing secrets into the cache.

Reply:
Too late :sweat:

Reply:
I will flush the cache

Reply:
Actually, the CI is not decoding the secrets!